src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -1062,6 +1072,12 @@ def test_aks_nodepool_create_scale_delete(self, resource_group, resource_group_l
             self.check('count', 3)
         ])
 
+        #nodepool update
+        self.cmd('aks nodepool update --resource-group={resource_group} --cluster-name={name} --name={nodepool2_name} --tags {new_tags}', checks=[
+            StringContainCheck("key2"),
+            StringContainCheck("value2")
```

Comment:
Minor problem: StringContainCheck is not strong enough

Reply:
I didn't find any useful function to check if string not contains some substring.

Reply:
hi. You can easily use `self.check('tag.key2','value2')` to check the response.

Reply:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md#sample-3-validate-the-return-json-value-using-jmespath